app/forms/new_phone_form.rb

```diff
@@ -38,7 +41,25 @@ def delivery_preference_sms?
   end
 
   def delivery_preference_voice?
-    VendorStatus.new.vendor_outage?(:sms)
+    VendorStatus.new.vendor_outage?(:sms) || setup_voice_preference?
+  end
+
+  # @return [Telephony::PhoneNumberInfo, nil]
+  def phone_info
+    return @phone_info if defined?(@phone_info)
+
+    if phone.blank? || !IdentityConfig.store.voip_check
```

Comment:
Given that we're now using `phone_info` for landline checking, I'm wondering if the "voip_check" name is accurate. Maybe we could rename it to `voip_or_landline_check` or `new_phone_type_check`?

Reply:
I don't follow. I see the `voip_check` as a separate process.

Reply:
Right, my concern is that because you also rely on this check for your new code, if the `voip_check` configuration is disabled, then the messaging for a landline phone will also be disabled, so the configuration name is now misleading (it's not just checking for VOIP numbers). I think we could improve it by renaming it to reflect that it also controls the new landline warning.

Reply:
I'd still like to see this configuration value renamed something like `phone_info_lookup`, but not a blocker if you want to create a follow-up task for it.